stdlib/Distributed/src/remotecall.jl

```diff
@@ -72,7 +72,7 @@ function test_existing_ref(r::AbstractRemoteRef)
         if isa(r, Future) && found.v === nothing && r.v !== nothing
             # we have recd the value from another source, probably a deserialized ref, send a del_client message
             send_del_client(r)
-            found.v = r.v
+            @atomic :sequentially_consistent found.v = r.v
```

Comment:
FYI, `sequentially_consistent` is the default so you don't need to mention in

```suggestion
            @atomic found.v = r.v
```

Reply:
Thanks for the note, I just decided to put it explicitly in case the default ever changes.
Do you think I should leave it explicitly or just rely on the default in this case?

Reply:
In all major languages, sequentially consistent is the default (and explicitly defined as such). I don't think it'll be changed in Julia. Also, it was decided to use `:sequentially_consistent` instead of "conventional" `seq_cst` although it's rather long to type since it's the default and so you don't have to put it everywhere.

Reply:
But maybe it makes sense to put it explicitly if you think weaker ordering does not work. If so, it'd be nice to leave the reasoning in the comment.

Reply:
Changing the default to be weaker would be breaking, so it won't happen.